docs/core/components/data-column.component.md

```diff
@@ -351,6 +354,40 @@ HTML `<data-column>` element example:
 </adf-tasklist>
 ```
 
+### Using the `keys` option
+
+`keys` options can be useful if your data object contains different properties and you would like to show them in one column:
+
+```json
+[
+    {"type": "text", "key": "name", "title": "name"},
+    {
+        "type": "text",
+        "key": "",
```

Comment:
why do u need to pass an empty key? is it mandatory ?

Reply:
Yes, the `key` is mandatory right now:
<img width="307" alt="Screenshot 2022-05-23 at 11 46 17" src="https://user-images.githubusercontent.com/13885344/169792498-42def5f5-ce39-4f8b-862d-e218852f6ef1.png">

Reply:
looks like we have key and keys that are in some way similar responsibility should we deprecate one ? or just make key accept string or array?

Reply:
Yes, I though about making the key `string | array`, the reason why I did not do it is that it could be possible breaking change (even I think it should be fine for e.g. alfresco-apps and only updates would be done in ng2)

Or for deprecation the `key`, when it will be time to remove it, we would have to update e.g. `lib/search/components/search-filter-container/search-filter-container.component.ts`

<img width="1111" alt="Screenshot 2022-05-24 at 09 20 41" src="https://user-images.githubusercontent.com/13885344/169974649-b81f2f14-972e-4f2b-a42c-6eccdee96db1.png">

to e.g. 
```
this.searchFilterQueryBuilder.getCategoryForColumn(this.col.keys[0]);
```

What do you think @mauriziovitale  @eromano? I am thinking more about the `string | array` approach, but do not have very strong opinion about it.

Reply:
from the other hand, maybe would be cleaner to have one type kor keys 🤔

Reply:
Will update the approach so we would use only `key` instead of `keys`